portafly/portafly_api.yml

```diff
@@ -7,35 +7,55 @@ info:
     A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API MGMT solution.
   license:
     name: Apache 2.0
-    url: https://www.apache.org/licenses/LICENSE-2.0
+    url: "https://www.apache.org/licenses/LICENSE-2.0"
```

Comment:
Is there a reason for enforcing quotes here?

Reply:
No, all the quotes were added by autoformatting :(

Reply:
Which autofformating?

Reply:
My editor or apicurio online editor, not sure